hive/src/main/java/io/confluent/connect/storage/hive/HiveConfig.java

```diff
@@ -155,26 +144,6 @@
           hiveIntegrationDependentsRecommender
       );
     }
-
-    {
-      // Define Schema configuration group
-      final String group = "Schema";
-      int orderInGroup = 0;
-
-      // Define Schema configuration group
-      CONFIG_DEF.define(
-          SCHEMA_COMPATIBILITY_CONFIG,
-          Type.STRING,
-          SCHEMA_COMPATIBILITY_DEFAULT,
-          Importance.HIGH,
-          SCHEMA_COMPATIBILITY_DOC,
-          group,
-          ++orderInGroup,
-          Width.SHORT,
-          SCHEMA_COMPATIBILITY_DISPLAY,
-          schemaCompatibilityRecommender
-      );
-    }
   }
 
   public static class SchemaCompatibilityRecommender extends BooleanParentRecommender {
```

Comment:
Is the intention to keep this class in `kafka-connect-storage-hive` even though it is also in `kafka-connect-storage-common` and the former depends on the latter, for backward compatibility reasons in case non-Confluent modules depend/use it?

Reply:
No, I just missed removing it. I feel that if we remove `HiveConfig.SCHEMA_COMPATIBILITY_CONFIG` we should remove this class too. The other option is to keep both with a note that we deprecate the ones here.

Reply:
WDYT?

Reply:
Sounds good.